applications/DG-Max/Algorithms/DGMaxDiscretization.h

```diff
@@ -116,6 +116,14 @@ class DGMaxDiscretization : public DGMaxDiscretizationBase {
     using TimeFunction = std::function<void(const PointPhysicalT&, double,
                                             LinearAlgebra::SmallVector<DIM>&)>;
 
+    /// Computed fields of the solution
+    struct Fields {
```

Comment:
Maybe, though this has the nice mirroring with the Div-version. I'm still thinking about adding another template parameter to SmallVector so that we can support all kinds of fixed size vectors.